### PR TITLE
fix: k8s security context permissions

### PR DESCRIPTION
**Pull request by krol3**
Fix issues related to k8s manifest in poste-ui https://github.com/aquasecurity/postee/issues/273

### CODE REVIEW

**Comment by sealneaward** [comment]
This lines up with the error I was having the other week. I can test these changes on my cluster to see if the error goes away.

**Comment by sealneaward** [comment]
The proposed changes work on my cluster adn fix the problem I was having.

File: deploy/kubernetes/postee.yaml [hunk 1]
```diff
@@ -154,11 +154,15 @@ spec:
           command:
             - /bin/chown
             - -R
-            - "1099"
-            - /server/database
+            - "1099:1099"
```
Comment (simar7) [inline deploy/kubernetes/postee.yaml:157]: What's 1099? I'm not familiar with it.

**Comment by simar7** [comment]
> The proposed changes work on my cluster and fix the problem I was having.

thanks @sealneaward for your feedback

File: deploy/kubernetes/postee.yaml (hunk 1 above)
Comment (simar7) [reply deploy/kubernetes/postee.yaml:157]: I found the answer - looks like we've been using this for Postee.

**Review by simar7 — APPROVED** [review]

**Pull request merged**